<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>230</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>230</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>580</width>
    <height>230</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="krfbIconLabel">
     <property name="minimumSize">
      <size>
       <width>128</width>
       <height>128</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>128</width>
       <height>128</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="rightLayout">
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>KDE Desktop Sharing</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="aboutLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>KDE Desktop Sharing allows you to grant permission to someone at a remote location for viewing and possibly controlling your desktop.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="enableSharingCheckBox">
       <property name="text">
        <string>Enable Desktop Sharing</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="detailsGroupBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="title">
        <string>Connection Details</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QFormLayout" name="addressFormLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="addressLabel">
            <property name="text">
             <string>Address</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QGridLayout" name="addressGridLayout">
            <item row="0" column="1">
             <widget class="QToolButton" name="addressAboutButton">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset theme="help-about">
                <normaloff/>
               </iconset>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="addressDisplayLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>127.0.0.1 : 5900</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="passwordFormLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="passwordLabel">
            <property name="text">
             <string>Password</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QGridLayout" name="passwordGridLayout">
            <item row="0" column="1">
             <widget class="QToolButton" name="passwordEditButton">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset theme="document-edit">
                <normaloff/>
               </iconset>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="passwordDisplayLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>TemporaryPassword</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>enableSharingCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>detailsGroupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
